src/common/com/intellij/plugins/haxe/model/HaxeImportModel.java

```diff
@@ -0,0 +1,154 @@
+  }
+
+  public boolean hasWildcard() {
+    return basePsi.getWildcard() != null;
+  }
+
+  @Nullable
+  public String getAliasName() {
+    HaxeIdentifier alias = basePsi.getAlias();
+    if (alias == null) return null;
+    return alias.getText();
+  }
+
+  public HaxeReferenceExpression getReferenceExpression() {
+    return basePsi.getReferenceExpression();
+  }
+
+  @NotNull
+  @Override
+  public List<HaxeModel> getExposedMembers() {
+    FullyQualifiedInfo qualifiedInfo = getQualifiedInfo();
+    List<HaxeModel> result = null;
+    if (hasWildcard()) {
+      if (qualifiedInfo.memberName != null) return Collections.emptyList();
+
+      if (qualifiedInfo.fileName != null && qualifiedInfo.className == null) {
+        qualifiedInfo = new FullyQualifiedInfo(qualifiedInfo.packagePath, qualifiedInfo.fileName, qualifiedInfo.fileName, null);
+      }
+      List<HaxeModel> items = HaxeProjectModel.fromElement(basePsi).resolve(qualifiedInfo, basePsi.getResolveScope());
+      if (items != null && items.size() > 0) {
+        result = items.stream()
+          .filter(model -> model instanceof HaxeExposableModel)
+          .flatMap(model -> ((HaxeExposableModel)model).getExposedMembers().stream())
+          .collect(Collectors.toList());
+      }
+    }
+    else {
+      if (hasAlias() && qualifiedInfo.fileName != null && qualifiedInfo.className == null) {
+        qualifiedInfo = new FullyQualifiedInfo(qualifiedInfo.packagePath, qualifiedInfo.fileName, qualifiedInfo.fileName, null);
+      }
+      result = HaxeProjectModel.fromElement(basePsi).resolve(qualifiedInfo, basePsi.getResolveScope());
+      if (result != null && !result.isEmpty() && (result.get(0) instanceof HaxeFileModel || result.get(0) instanceof HaxePackageModel)) {
+        result = ((HaxeExposableModel)result.get(0)).getExposedMembers();
+      }
+    }
+
+    return result != null ? result : Collections.emptyList();
+  }
+
+  @Nullable
+  public PsiElement exposeByName(String name) {
+    if (name == null) return null;
+
+    if (hasWildcard()) {
+      for (HaxeModel exposedMember : getExposedMembers()) {
+        if (Objects.equals(exposedMember.getName(), name)) return exposedMember.getBasePsi();
+      }
+    }
+    else {
+      if (getReferenceExpression() != null) {
+        FullyQualifiedInfo qualifiedInfo = getQualifiedInfo();
+
+        if (equalsToAlias(name) || qualifiedInfo.equalsToNamedPart(name) || getExposedMember(name) != null) {
+          HaxeModel result = getExposedMember(name);
+          return result != null ? result.getBasePsi() : null;
+        }
+      }
+    }
+
+    return null;
+  }
+
+  private boolean equalsToAlias(String name) {
+    return hasAlias() && Objects.equals(getAliasName(), name);
+  }
+
+  private HaxeModel getExposedMember(String name) {
+    List<HaxeModel> members = getExposedMembers();
+    if (members.isEmpty()) return null;
+    if (hasAlias()) {
+      return (Objects.equals(getAliasName(), name)) ? members.get(0) : null;
```

Comment:
I'm not understanding why we still don't have a search for the proper exposed member.  An alias can be for any class in the imported file, right?  So, if the alias matches, we should swap the alias for the underlying name and continue the search.  What am I missing?